ospf6d/ospf6_top.h

```diff
@@ -230,7 +230,7 @@ struct ospf6 {
 
 #define OSPF6_EXTL_AGGR_DEFAULT_DELAY 5
 	/* For ASBR summary delay timer */
-	int aggr_delay_interval;
+	unsigned int aggr_delay_interval;
```

Comment:
This change is in the commit named `ospfd: a few minor fixes on ospfd cli`.
Please, move it into a separate commit which should start with `ospf6d:`.

Reply:
Thanks for review！
Already repushed.